Week2/homework/maartjes_work.js

```diff
@@ -1,47 +1,54 @@
-'use strict';
+"use strict";
 
 const monday = [
   {
-    name: 'Write a summary HTML/CSS',
+    name: "Write a summary HTML/CSS",
     duration: 180
   },
   {
-    name: 'Some web development',
+    name: "Some web development",
     duration: 120
   },
   {
-    name: 'Fix homework for class10',
+    name: "Fix homework for class10",
     duration: 20
   },
   {
-    name: 'Talk to a lot of people',
+    name: "Talk to a lot of people",
     duration: 200
   }
 ];
 
 const tuesday = [
   {
-    name: 'Keep writing summary',
+    name: "Keep writing summary",
     duration: 240
   },
   {
-    name: 'Some more web development',
+    name: "Some more web development",
     duration: 180
   },
   {
-    name: 'Staring out the window',
+    name: "Staring out the window",
     duration: 10
   },
   {
-    name: 'Talk to a lot of people',
+    name: "Talk to a lot of people",
     duration: 200
   },
   {
-    name: 'Look at application assignments new students',
+    name: "Look at application assignments new students",
     duration: 40
   }
 ];
 
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+let hourSalary = 25;
+const earnings = tasks
+  .map(x => x.duration / 60) // transforming all task durations to hours
```

Comment:
I see you have put comments in there to explain what is happening. Nonetheless, you have not made the variables descriptive. I would advice you to make the variables descriptive, and to minimize the amount of comments in there. Additionally, aim to have a variable name that  is that descriptive that no comments are required . It requires less reading and there is less code you have to maintain. Example: tasks.map(minutesToHours => minutesToHours.duration / 60)

Reply:
Thank you Sil for your feedback,
I've made the variables more descriptive, but kept the comments cause they help me understand what's going on. if that's okay of course.